relayer/__init__.py

```diff
@@ -7,17 +7,45 @@
 
 
 class Relayer(object):
+    """
+    Arguments:
+        logging_topic: desired kafka topic to send logs, this will get modified if a prefix o suffix is set.
+        context_handler_class: class that receives an emitter and expose at least a method emit and log.
+    Keyword Arguments:
+        kafka_hosts: 'host[:port]' string (or list of 'host[:port]'
+            strings) that the producer should contact to bootstrap initial
+            cluster metadata. This does not have to be the full node list.
+            It just needs to have at least one broker that will respond to a
+            Metadata API Request.
+        topic_prefix (str): value to prefix all topics handled by relayer.
+            Defaults to empty string.
+        topic_suffix (str): value to suffix all topics handled by relayer.
+            Defaults to empty string.
+        source: If defined it must be a json serializable value.
+            Defaults to topic_prefix + logging_topic + topic_suffix.
+        producer_opts (dict): optional dictionary with the configuration
+            for http://kafka-python.readthedocs.io/en/master/apidoc/KafkaProducer.html
+    """
 
-    def __init__(self, logging_topic, context_handler_class, kafka_hosts=None, topic_prefix='', topic_suffix='', source=''):
+    def __init__(self, logging_topic, context_handler_class, **kwargs):
         self.logging_topic = logging_topic
-        if not kafka_hosts:
+
+        if 'kafka_hosts' not in kwargs:
```

Comment:
I'm fine with using `**kwargs` but then we need to maintain these hardcoded strings, which are pretty fragile
one idea that comes to mind is using constants but I'm not a big fan
any other ideas?